src/Storages/Streaming/StorageRandom.cpp

```diff
@@ -556,9 +576,16 @@ void registerStorageRandom(StorageFactory & factory)
         {
             storage_random_settings->loadFromQuery(*args.storage_def);
         }
-
+        if (storage_random_settings->eps.value == 0 || storage_random_settings->interval_time.value == 0 
```

Comment:
When user don't set an `eps` or the `eps = 0`, it shall mean no limit (generate as fast as possible) ? This shall be a valid configuration ?

Reply:
In issue #72, Jove suggested that I set the default value to 1000.I need to check  if there will be memory related issues if there is no limit.